documentation/3.3/content/userguide/managing-domains/domain-lifecycle/liveness-readiness-probe-customization.md

```diff
@@ -15,18 +15,21 @@ This document describes how to customize the liveness and readiness probes for W
 
 #### Liveness probe customization
 
-The liveness probe is configured to check that a server is alive by querying the Node Manager process.  By default, the liveness probe is configured to check liveness every 45 seconds and to timeout after 5 seconds.  If a pod fails the liveness probe, Kubernetes will restart that container.
+The liveness probe is configured to check that a server is alive by querying the Node Manager process.  By default, the liveness probe is configured to check liveness every 45 seconds, to timeout after 5 seconds, and to perform the first check after 30 seconds.  The default success and failure threshold values are 1.  If a pod fails the liveness probe, Kubernetes will restart that container.
 
-You can customize the liveness probe interval and timeout using the `livenessProbe` attribute under the `serverPod` element of the domain resource. 
+You can customize the liveness probe initial delay, interval, timeout and failure threshold using the `livenessProbe` attribute under the `serverPod` element of the domain resource.
```

Comment:
Serial (Oxford) comma needed after timeout, here and in the next sentence, and once more later in the document.